### DISCUSSION TITLE
What is the purposeof replacing "<unknown id>" with "id"?

### DISCUSSION THREAD

**Discussion by jan-glx** [pydna-group/pydna#85] [Q&A]
https://github.com/BjornFJohansson/pydna/blob/75fa7e2c783eabfa32f580d7c0ea2620f907390b/src/pydna/seqrecord.py#L630

**Comment by BjornFJohansson** [comment]
Hi @jan-glx 
I think the <unknown id> is so very ugly...

**Reply by jan-glx** [reply]
the `"<unkown id>"`? Shouldn't it rather be `None` with the printing function taking care of displaying that in a sensible way. Might break some code though.  What would you think about `answer.name = "-".join((name for name in (self.name, other.name) if name is not None))`?
